getting-started/templates/pilot-sizing.yaml

```diff
@@ -136,6 +136,16 @@ dataframeservice:
     targetMemoryUtilizationPercentage: 70
 
   ingestion:
+    # Configuration for ingestion requests that use the Apache Arrow format.
+    arrow:
+      # The maximum number of bytes that the server will allocate in order to hold a single record
```

Comment:
Suggestion:
'Allocates the maximum number of bytes for the server to hold for a single record batch during an ingestion request. This value must be greater than zero. Larger values require additional pod memory per request.

For non-compressed requests, the system compares this value to the size of the record batch in memory. For compressed requests, the system compares this value to the size of the uncompressed record batch in memory. Also included is the size of each compressed value buffer.'